src/primitives/transaction.h

```diff
@@ -295,6 +297,9 @@ inline void UnserializeTransaction(TxType& tx, Stream& s) {
     unsigned char flags = 0;
     tx.vin.clear();
     tx.vout.clear();
+    tx.witness.SetNull();
+    const_cast<CTxWitness*>(&tx.witness)->vtxinwit.resize(tx.vin.size());
```

Comment:
wrong place, will always resize to 0... move to right before `g_con_elementswitness` check